composer.json

```diff
@@ -36,15 +36,20 @@
     "the-events-calendar/coding-standards": "dev-master",
     "the-events-calendar/tec-testing-facilities": "dev-master",
     "vlucas/phpdotenv": "^4.1",
-    "wp-coding-standards/wpcs": "^2.1"
+    "wp-coding-standards/wpcs": "^2.1",
+    "phpstan/phpstan": "^1.10",
+    "phpstan/phpstan-deprecation-rules": "^1.1",
+    "phpstan/extension-installer": "^1.3",
+    "szepeviktor/phpstan-wordpress": "*"
   },
   "config": {
     "preferred-install": "dist",
     "platform": {
-      "php": "7.3.33"
+      "php": "7.4"
```

Comment:
Our min required PHP version is 7.4. Not sure if there is a good reason to set this lower?